src/frontend/Sider/Behandling/Vilkårvurdering/Begrunnelse.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import { FC } from 'react';
+
+import { Textarea } from '@navikt/ds-react';
+
+import { BegrunnelseRegel, Regel } from '../../../typer/regel';
+import { Vurdering } from '../vilkår';
+
+interface Props {
+    svar: Vurdering;
+    regel: Regel;
+    onChange: (tekst: string) => void;
+}
+
+const Begrunnelse: FC<Props> = ({ svar, onChange, regel }) => {
+    const begrunnelseType = svar.svar && regel.svarMapping[svar.svar].begrunnelseType;
+    const skjulBegrunnelse = (begrunnelseType ?? BegrunnelseRegel.UTEN) === BegrunnelseRegel.UTEN;
```

Comment:
Skjønner ikke hvorfor `??`er brukt her 🤔  Gir samme effekt, men synes den blir vanskeligere å lese
```suggestion
    const skjulBegrunnelse = (begrunnelseType || BegrunnelseRegel.UTEN) === BegrunnelseRegel.UTEN;
```